### PR TITLE
Update to BA plugin:

### PR DESCRIPTION
**Pull request by Jacoblairm**
- Fixed this: https://i.imgur.com/APUYSyL.jpg by nulling the GameTimer when the game is complete.
- Added an option (by popular demand) that shows the round time (in seconds) instead of the call change time (off by default).
- Fix a rounding issue with the timer, by rounding Duration.toMillis() instead of just using the Duration.getSeconds() (which was always rounded down)

### CODE REVIEW

**Comment by deathbeam** [comment]
Maybe you should 1. split commits to commit fixing bug and commit adding new feature, 2. display both timers and not one or another if both are enabled?

**Comment by Jacoblairm** [comment]
So having them side by side if they're both enabled like this  -  '00:12 (18') or '(18) 00:12' ?

File: runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultConfig.java [hunk 1]
```diff
@@ -42,6 +42,16 @@ default boolean showTimer()
 		return true;
 	}
 
+	@ConfigItem(
+		keyName = "secondsTimer",
+		name = "Show elapsed seconds",
+		description = "Instead of showing the call change timer, the wave elapsed time will be shown (in seconds)"
```
Comment (MagicfTail) [inline runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultConfig.java]: This description needs to be changed now that you no longer remove the other timer

Comment (Jacoblairm) [reply runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultConfig.java]: Ah yea good point

**Pull request closed without merging**